README.md

```diff
@@ -47,113 +47,163 @@ If you find `googler` useful, please consider donating via PayPal.
-<pre>Enter 'n', 'p', 'g keywords', or result number to continue: <b>g n</b>
-Enter 'n', 'p', 'g keywords', or result number to continue: <b>g g keywords</b>
-Enter 'n', 'p', 'g keywords', or result number to continue: <b>g 1984</b></pre>
-Note that Google ignores searches for negative numbers (e.g. `-1984`).
-12. Input and output <b>redirection</b>:
-<pre>$ googler -C hello world < input > output</pre>
-Note that `-C` is required to avoid printing control characters. `2>&1` would error as the console geometry is calculated from `stderr`.
-13. <b>Piping</b> `googler` output:
-<pre>$ googler -C hello world | tee output</pre>
-14. More <b>help</b>:
-<pre>$ googler
-$ man googler</pre>
+1. Google **hello world**:
+
+        $ googler hello world
+
+2. Fetch **15 results** updated within last **14 months**, starting from the **3<sup>rd</sup> result** for the string **cmdline utility** in **site** tuxdiary.com:
+
+        $ googler -n 15 -s 3 -t m14 cmdline utility site:tuxdiary.com
+
+3. Read recent **news** on gadgets:
+
+        $ googler -N gadgets
+
+4. Fetch results on IPL cricket from **Google India** server in **English**:
+
+        $ googler -c in -l en IPL cricket
+
+5. Search quoted text e.g. **it's a "beautiful world" in spring**:
+
+        $ googler it\'s a \"beautiful world\" in spring
+
+6. Search for a **specific file type**:
+
+        $ googler instrumental filetype:mp3
+
+7. Disable **automatic spelling correction**, e.g. fetch results for `googler` instead of `google`:
+
+        $ googler -x googler
+
+8. **I'm feeling lucky** search:
+
+        $ googler -j leather jackets
+
+9. **Website specific** search alias:
+
+        alias t='googler -n 7 site:tuxdiary.com'
+
+10. Alias to find **definitions of words**:
+
+        alias define='googler -n 2 define'
+
+11. Look up `n`, `p`, `g co` or a number at the **navigation prompt**: As the navigation prompt recognizes `n`, `p`, `g` or numbers as commands, you need to prefix them with `g`, e.g., `g n`, `g g keywords`, `g 1984`.
+
+12. Input and output **redirection**:
+
+        $ googler -C hello world < input > output
+
+    Note that `-C` is required to avoid printing control characters (for colored output).
+
+13. **Piping** `googler` output:
+
+        $ googler -C hello world | tee output
+
+14. More help:
+
+        $ googler
+        $ man googler
 
 # Troubleshooting
 
-1. If print() throws the following error complaining about handling Unicode with `ascii` codec:
-<pre>UnicodeEncodeError: 'ascii' codec can't encode character '\u201c' in position 0: ordinal not in range(128)</pre>
-add the following to your `~/.bashrc`:
-<pre>export PYTHONIOENCODING=UTF-8</pre>
-If you use fish shell, add the following in `~/.config/fish/config.fish`:
-<pre>set -x PYTHONIOENCODING UTF-8</pre>
-Ref issue [#21](https://github.com/jarun/googler/issues/21).
+1.  If `print()` throws the following error complaining about handling Unicode with `ascii` codec:
```

Comment:
This is intentional, so that this paragraph lines with the following paragraphs under the same list item. Four-space indents are required for some Markdown parsers, a rather unfortunate fact (CommonMark, a Markdown standardization effort, which I'm an advocate for, removes that annoying restriction and only requires you to indent enough to be unambiguous, but the semistandard is not recognized accross the board).